geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+# Bond angles:  theta0 (degrees)
+# Source Paper: Min J, Bucchino MP, Kilchenstein KM, Ziurys LM (2016) Rotational spectroscopy of ClZnCH3 (X1
+A1): Gasphase synthesis and characterization of a monomeric Grignard-type reagent. Chem Phys Lett 646(4):174-178
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Zn Cl  2.23(7)
+Zn C   1.9201
+C  H   1.105
+Zn C H 110.2(5)
+H  C H 108.7
+
+# Zn C 1.9201, C H 1.105 and H C H 108.7 fixed to the experimental value of CH3ZnI
+
+NAME = CH3IZn 2014
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Bucchino MP, Young JP, Sheridan PM, Ziurys LM (2014) Structural determination and gas-phase synthesis of
+monomeric, unsolvated IZnCH3 (X1A1): a model organozinc halide. J Phys Chem A 118(47):11204-11210
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Zn I   2.4076(2)
+Zn C   1.9201(2)
+C  H   1.105(9)
+Zn C H 110.2(5)
+H  C H 108.7(5)
+
+NAME = CH3NOS
+# Point Group: C3v
+# Shape: Linear --- Trigonal pyramidal
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Liu X, Xu Y (2011) Infrared and microwave spectra of the acetylene-ammonia and carbonyl sulfide-ammonia
+complexes: a comparative study of a weak C–H⋅⋅⋅N hydrogen bond and an S⋅⋅⋅N bond. Phys Chem Chem Phys
+13(31):14235-14242
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+S...N   3.3233(2)
+NH3 SCO 4.3607(2)
+α       25.23(4)
+
+# α: Angular oscillation of the ammonia subunit. 
+
+NAME = CH3O 
+# Point Group: C3v
+# Shape: Seesaw
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Liu J, Chen MW, Melnik D, Miller TA, Endo Y, Hirota E (2009) The spectroscopic characterization of the
+methoxy radical. II. Rotationally resolved A2A1-X2E electronic and X2E microwave spectra of the perdeuteromethoxy radical CD3O. J Chem Phys 130(7):074303/1-074303/11
+https://doi.org/10.1063/1.2955739
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C O   1.36039(8)
+C H   1.10697(9)
+H C H 107.76(1)
+
+NAME = CH4Zn 2010
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Flory MA, Apponi AJ, Zack LN, Ziurys LM (2010) Activation of methane by zinc: Gas-phase synthesis,
+structure, and bonding of HZnCH3. J Amer Chem Soc 132(48):17186-17192
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+H  Zn  1.5209(1)
+Zn C   1.9281(2)
+C  H   1.140(9)
+Zn C H 110.2(3)
+H  C H 108.7(3)
+
+NAME = C2AgCl  2015
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Zaleski DP, Tew DP, Walker NR, Legon AC (2015) Chemistry in laser-induced plasmas: formation of M-C≡CCl (M = Ag or Cu) and their characterization by rotational spectroscopy. J Phys Chem A 119(12):2919-2925
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Ag   C  2.015(14)
+C  ≡ C  1.2219
+C    Cl 1.635(6)
+Ag...Cl 4.8722(2)
```

Comment:
2.015+1.2219+1.635 = 4.8719 so I'm confused about the meaning of 4.8722 here.

Reply:
Could it be that when they measured the length between Ag and Cl it was subject to error because the molecules was not perfectly linear?

Reply:
I think I need to look at how the length of 4.8722(3) for Ag...Cl was determined.